broker-core/src/test/java/io/zeebe/broker/workflow/timer/TimerCatchEventTest.java

```diff
@@ -84,35 +97,36 @@
           .endEvent()
           .done();
 
-  public EmbeddedBrokerRule brokerRule = new EmbeddedBrokerRule();
-  public ClientApiRule apiRule = new ClientApiRule(brokerRule::getClientAddress);
-
-  @Rule public RuleChain ruleChain = RuleChain.outerRule(brokerRule).around(apiRule);
-
   private PartitionTestClient testClient;
 
   @Before
   public void init() {
     testClient = apiRule.partitionClient();
   }
 
+  @After
+  public void tearDown() {
+    brokerRule.getClock().reset();
+  }
+
   @Test
   public void testLifeCycle() {
     // given
+    final String processId = Strings.newRandomValidBpmnId();
     final BpmnModelInstance workflow =
-        Bpmn.createExecutableProcess(PROCESS_ID)
+        Bpmn.createExecutableProcess(processId)
```

Comment:
I think it is not necessary to generate a process id. To be safe, we could use the workflow key.